README.md

```diff
@@ -25,6 +25,14 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 
+## Testing
+
+1. Place `personium-ex-mailsender` jar file and `Ext_MailSender.properties` file in `/personium/personium-engine/`
+1. Launch prequire software with `docker-compose up`
+1. Launch tomcat9 locally
```

Comment:
```suggestion
1. Launch tomcat9 locally  
`systemctl start tomcat`
```

Reply:
It is more friendly to provide a sample command. Please use the correct tomcat service name when you reflect on the change.